core/domain/user_services.py

```diff
@@ -896,6 +898,71 @@ def _update_user_contribution_rights(user_contribution_rights):
         remove_contribution_reviewer(user_contribution_rights.id)
 
 
+def _update_reviewer_counts_in_community_contribution_stats_transactional(
+        future_user_contribution_rights):
+    """Updates the reviewer counts in the community contribution stats based
+    on the updates to the given user contribution rights.
```

Comment:
Maybe clarify that these are specifically future updates.

Reply:
Added some more wording for that to clarify